client/stylesheets/_toCat.scss

```diff
@@ -3879,7 +3886,9 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .footerLeft {
-    margin: 0 15% 0 15%;
+    margin: 0 auto;
+    width: 80%;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered margin, max-width, width